packages/nextjs-experimental/tailwind.config.js

```diff
@@ -7,10 +7,24 @@ module.exports = {
   ],
   theme: {
     extend: {
+      fontFamily: {
+        sans: ['Source Sans Pro', 'sans-serif'],
+      },
       backgroundImage: {
         'gradient-radial': 'radial-gradient(var(--tw-gradient-stops))',
         'gradient-conic': 'conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))',
       },
+      colors: {
```

Comment:
Fixing colors not showing up: https://fixie-ai.slack.com/archives/C05B6EE9JE5/p1687460043947939

Reply:
Nice!